tests/perf.c

```diff
@@ -358,55 +351,79 @@ setup_running_cached(uint32_t count, struct test_state *state)
     }
     lyd_free_siblings(data);
 
+    /* get data, so that if connection is cached, cache gets filled */
+    if ((r = sr_get_data(state->sess, "//.", 0, 0, 0, &retr_data))) {
+        return r;
+    }
+    sr_release_data(retr_data);
```

Comment:
Should not be needed on the current *sysrepo* version.

Reply:
done